doctests/dt-json.js

```diff
@@ -0,0 +1,382 @@
+// EXAMPLE: json_tutorial
+// HIDE_START
+import assert from 'assert';
+import {
+  createClient
+} from 'redis';
+
+const client = await createClient();
+await client.connect();
+// HIDE_END
+// REMOVE_START
+await client.flushDb();
+// REMOVE_END
+
+// STEP_START set_get
+const res1 = await client.json.set("bike", "$", '"Hyperion"');
+console.log(res1); // OK
+
+const res2 = await client.json.get("bike", "$");
+console.log(res2); // "Hyperion"
+
+const res3 = await client.json.type("bike", "$");
+console.log(res3); //  [ 'string' ]
```

Comment:
testing against v5, and in v5, this returns it outside of array syntax.  need to understand what's going on.